com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/util/ReadOnlyParametersCheckerTest.java

```diff
@@ -0,0 +1,103 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.AUTHORIZATION_URL;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.CONTROLLER_URL;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_APP_NAME;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_CONTAINER_NAME;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_DOMAIN;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_HOST;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_IDLE_DOMAIN;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_IDLE_HOST;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_IDLE_URI;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_IDLE_URL;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_INSTANCES;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_SERVICE_NAME;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_URI;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_URL;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.DEFAULT_XS_APP_NAME;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.GENERATED_PASSWORD;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.GENERATED_USER;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.ORG;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.PROTOCOL;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.SPACE;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.TIMESTAMP;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.USER;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.XS_TYPE;
+
+import java.text.MessageFormat;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import com.sap.cloud.lm.sl.cf.process.message.Messages;
+import com.sap.cloud.lm.sl.common.SLException;
+import com.sap.cloud.lm.sl.mta.handlers.v3.DescriptorParser;
+import com.sap.cloud.lm.sl.mta.model.DeploymentDescriptor;
+
+public class ReadOnlyParametersCheckerTest {
```

Comment:
I think it is ok to validate only for one read-only parameter per group.
Now it is harder for read.

Reply:
Done, only one parameter per group is tested